src/DeviceInterface.js

```diff
@@ -260,8 +260,9 @@ class AppleDeviceInterface extends InterfacePrototype {
 
         this.refreshAlias = () => wkSend('emailHandlerRefreshAlias')
 
-        this.isDeviceSignedIn = async () => {
+        this._checkDeviceSignedIn = async () => {
             const {isAppSignedIn} = await wkSendAndWait('emailHandlerCheckAppSignedInStatus')
+            this.isDeviceSignedIn = () => !!isAppSignedIn
```

Comment:
❓ Just confirming we're ok with this device interface using the inherited `isDeviceSignedIn` method (which returns `undefined`) until this async code is run?

Reply:
Yes. We could override it locally to return `false` explicitly but I'd think it just adds noise.